wp-content/themes/twentyseventeen-wp20/page-news.php

```diff
@@ -0,0 +1,40 @@
+<?php get_header(); ?>
+
+<div id="primary" class="content-area">
+	<main id="main" class="site-main" role="main">
+
+		<h1 class="entry-title screen-reader-text">
+			<?php the_title(); ?>
+		</h1>
+		
+		<?php
+		$posts = get_posts( array(
+			'posts_per_page' => -1,
+			'post_type'      => 'post',
+			'post_status'    => 'publish',
+			'orderby'        => 'date',
+			'order'          => 'DESC',
+		) );
+		
+		$posts_by_month = array();
+		foreach ( $posts as $post ) {
+			$month = date( 'F', strtotime( $post->post_date ) );
+			$month = substr( $month, 0, 3 );
+			$posts_by_month[ $month ][] = $post;
+		}
```

Comment:
I've refined this, but the initial suggestion for how to query and group was suggested by ChatGPT